### PR TITLE
chore: Print HTPasswd user credentials

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

### What does this PR do?
Print HTPasswd user credentials when Eclipse Che provisions OpenShift OAuth user.

### Screenshot/screencast of this PR
```
    ✔ -------------------------------------------------------------------------------
    ✔ Users Dashboard           : https://che-eclipse-che.apps.abazko.devcluster.openshift.com
    ✔ HTPasswd user credentials : "che:O3KIan".
```

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18152

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: src/tasks/che.ts [hunk 1]
```diff
@@ -675,7 +675,20 @@ export class CheTasks {
 
           const cheUrl = await this.che.cheURL(flags.chenamespace)
           messages.push(`Users Dashboard           : ${cheUrl}`)
-          messages.push('Admin user login          : "admin:admin". NOTE: must change after first login.')
+
+          const cr = await this.kube.getCheCluster(flags.chenamespace)
+          if (ctx.isOpenShift && cr && cr.spec && cr.spec.auth && cr.spec.auth.openShiftoAuth) {
```
Comment (AndrienkoAleksandr) [inline src/tasks/che.ts:680]:
```suggestion
          if (ctx.isOpenShift4 && cr && cr.spec && cr.spec.auth && cr.spec.auth.openShiftoAuth) {
```

Comment (tolusha) [reply src/tasks/che.ts:680]: Agreed

Comment (tolusha) [reply src/tasks/che.ts:680]: No.
The main point is to print `admin:admin` only of OAuth isn't enable and HTPasswd user credentials will be printed only of secret is created.

**Review by AndrienkoAleksandr — APPROVED** [review]

**Review by flacatus — APPROVED** [review]

**Pull request merged**